app/lib/smooch_nlu.rb

```diff
@@ -46,13 +46,13 @@ def update_keywords(language, keywords, keyword, operation, doc_id, context)
       keywords << keyword
       alegre_operation = 'post'
       alegre_params = common_alegre_params.merge({ text: keyword, models: ALEGRE_MODELS_AND_THRESHOLDS.keys })
+      Bot::Alegre.get_sync_raw_params(alegre_params, "text") if alegre_operation && alegre_params
     elsif operation == 'remove'
       keywords -= [keyword]
       alegre_operation = 'delete'
       alegre_params = common_alegre_params.merge({ quiet: true })
+      Bot::Alegre.request_delete_from_raw(alegre_params, "text") if alegre_operation && alegre_params
```

Comment:
This can be simplified further, it seems... you don't need `alegre_operation` anymore, since the calls were moved to the condition blocks... right?

Reply:
you're right! Updated.